docs/appendices/examples.md

```diff
@@ -56,30 +56,17 @@ EGLEvent getEvent() {
 then you can destructure it and get the values using a tuple
 
 ```pony
+use @getEvent[EGLEvent]()
 type EGLEvent is (U8, F32, F32)
-(var code, var x, var y) = @getEvent[EGLEvent]()
-```
-
-## Get and pass pointers to FFI
-
-```pony
-primitive _XDisplayHandle
-primitive _EGLDisplayHandle
-
-let x_dpy = @XOpenDisplay[Pointer[_XDisplayHandle]](U32(0))
-if x_dpy.is_null() then
-  env.out.print("XOpenDisplay failed")
-end
-
-let e_dpy = @eglGetDisplay[Pointer[_EGLDisplayHandle]](x_dpy)
-if e_dpy.is_null() then
-  env.out.print("eglGetDisplay failed")
-end
```

Comment:
is this an intentional removal @ergl ?

Reply:
Yes, this specific part is already covered here https://tutorial.ponylang.io/c-ffi/calling-c.html#get-and-pass-pointers-to-ffi, so it feels like a duplicate

Reply:
Gotcha. Looks reasonable.